lisa/conf.py

```diff
@@ -1235,6 +1243,62 @@ def from_map(cls, *args, **kwargs):
     def to_map(self):
         return dict(self._get_effective_map())
 
+    def to_yaml_map(self, path, add_placeholder=False, placeholder='<no default>'):
+        """
+        Write a configuration file, with the key descriptions in comments.
+
+        :param path: Path to the file to write to.
+        :type path: str
+
+        :param add_placeholder: If ``True``, a placeholder value will be used
+            for keys that don't have values. This allows creating template
+            configuration files that list all keys.
+        :type add_placeholder: bool
+
+        :param placeholder: Placeholder to use for missing values when
+            ``add_placeholder`` is used.
+        :type placeholder: object
+        """
+
+        def format_comment(key_desc):
+            comment = key_desc.help
+            if not comment:
+                return comment
+            else:
+                return comment[0].upper() + comment[1:]
```

Comment:
Is that just a capitalization of the first letter? If so, wouldn't `return comment.title()` suffice?

Reply:
title() makes all first letters uppercase, not just for the first word:
```
>>> 'foo bar'.title()
'Foo Bar'
```

Reply:
Aaah. What about `.capitalize()`? :D

Reply:
Good point, the docstring of `capitalize()` is a kind of tautology ...